e2e/tests/functional/recentObjects.e2e.spec.js

```diff
@@ -50,14 +54,9 @@ test.describe('Recent Objects', () => {
         await page.mouse.move(0, 100);
```

Comment:
I can't leave a comment on line #48 but do we have a better locator for this element?

Reply:
updated to not use `nth-child`. A bit more deterministic